stdlib/Unicode/docs/src/index.md

```diff
@@ -0,0 +1,25 @@
+# Unicode
+
+```@docs
+Unicode.is_assigned_char
+Unicode.normalize_string
```

Comment:
How about renaming `is_assigned_char` to `isassigned` and generalizing it to strings the same way the rest are – i.e. by `all` reduction over the characters in the string. Similarly, we could rename `normalize_string` to `normalize` now that it's in the `Unicode` module.

Reply:
I've added them in the commit deprecating `isnumber` in favor of `isnumeric`. Though note that the names clash with corresponding functions in Base, so they always need to be qualified. I guess that's OK.

`isassigned` was already restricted to `Char` arguments. But I noticed `isascii` accepts strings directly, which is inconsistent with other functions. Time to deprecate it too?